compiler/src/typed/builtin_types.re

```diff
@@ -95,6 +99,8 @@ and type_option = var =>
 and type_result = (ok, err) =>
   newgenty(TTyConstr(path_result, [ok, err], ref(TMemNil)))
 and type_list = var => newgenty(TTyConstr(path_list, [var], ref(TMemNil)))
+and type_range = var =>
+  newgenty(TTyRecord([("rangeStart", var), ("rangeEnd", var)]))
```

Comment:
It's insane for me to suggest this right?
```suggestion
  newgenty(TTyRecord([(Ident.name(ident_range_start), var), (Ident.name(ident_range_end), var)]))
```

Reply:
I don't know the difference. I thought this type took a string?

Reply:
Oh, I read that wrong, it's pulling the string name from the ident